dev-docs/source/Testing/ErrorReporter-ProjectRecovery/ErrorReporterTesting.rst

```diff
@@ -28,78 +28,110 @@ Error Reporter test
+- Click the `Show More Details` button to open the Show More Details dialog. This should show user details such as OS.
+  There should be a python stacktrace as this unhandled exception occurred in Python. Close this dialog.
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
+- This time, UNTick the `Remember Me` checkbox
 - Make sure the `Continue` radio button is checked
-- Click the `Share non-identifiable information` box
+- Click the `Share non-identifiable information` button
 - You should be returned to the main Mantid window
-- Check with the database admin
+- Check with the database admin that an error report was sent withOUT a name, email or stacktrace, BUT WITH a textbox.
 
 ---------------
 
-5. Open MantidWorkbench
+5. MantidWorkbench is still open
 
 - Load the file `Training_Exercise3a_SNS.nxs`
 - Run the `NormaliseToMonitor` algorithm
-- This should cause an error box saying Mantid has thrown an unexpected exception
-- Make sure the hyperlink to the privacy policy works
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
+- Contact details from before should NOT be displayed and the `Remember Me` checkbox NOT ticked
 - Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
-- In the `Name` box enter `Public2` and in the `Email` box enter `public2`
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
+- Leave the `Name` box EMPTY and in the `Email` box enter `public2`
+- Tick the `Remember Me` checkbox
 - Make sure the `Continue` radio button is checked
-- Click the `Yes, share information` box
+- Click the `Yes, share information` button
 - You should be returned to the main Mantid window
-- Check with the database admin
+- Check with the database admin that an error report was sent WITH a name, email, stacktrace and a textbox.
 
 ---------------
 
-6. Open MantidWorkbench
+6. MantidWorkbench is still open
 
 - Load the file `Training_Exercise3a_SNS.nxs`
 - Run the `NormaliseToMonitor` algorithm
-- This should cause an error box saying Mantid has thrown an unexpected exception
-- Make sure the hyperlink to the privacy policy works
+- This should cause an error reporter dialog saying Mantid has thrown an unexpected exception
+- Only the `public2` email from Test 5 should be displayed in the email box and the `Remember Me` checkbox ticked.
+  Make sure the `Share non-identifiable information` button is currently enabled (not greyed out)
 - Enter some text in the `Name` box; make sure the `Share non-identifiable information` button gets greyed out
 - In the `Name` box enter `Public3` and in the `Email` box enter `public3`
+- Input some additional information into the main textbox. Try to include characters that need to be escaped such as `"`
```

Comment:
This might be better with double backticks to format it more like `"` than *"*

Reply:
Need to use \``"`` for this to work, thanks for spotting!